pkg/kubefedctl/orphaning/orphaning.go

```diff
@@ -90,7 +90,11 @@ func (o *orphanResource) Complete(args []string) error {
 	}
 	o.resourceName = args[1]
 
-	return nil
+	var err error
+	if len(o.resourceNamespace) == 0 {
+		o.resourceNamespace, err = util.GetNamespace(o.HostClusterContext, o.Kubeconfig, config)
```

Comment:
Please localize the error check since the error doesn't need to be used outside of the block e.g.

```golang
if ... {
  o.resourceNamespace, err := ..
  return err
}
return nil
```

Reply:
Done